Website can't be started locally

**Describe the bug**
Error starting vault site.

**To Reproduce**
1. Clone the repo
2. cd website && make website

**Expected behavior**
Website to start?

**Environment:**
Linux <host> 4.15.0-36-generic #39-debian SMP Mon Sep 24 16:19:09 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux


**Additional context**
It'd be also nice if docker the used docker image (netlify:build) doesn't take too much space. 3.8 gb to start a website seems too much.

Log output:

```
Digest: sha256:5ffe665bf281e6ef428d8fa96d3e7d4853411b6f72ff62356c652175ab8ff416
Status: Downloaded newer image for netlify/build:latest
mkdir: cannot create directory ‘/opt/buildhome/cache/node_version’: Permission denied
mkdir: cannot create directory ‘/opt/buildhome/cache/ruby_version’: Permission denied
mkdir: cannot create directory ‘/opt/buildhome/cache/node_modules’: Permission denied
mkdir: cannot create directory ‘/opt/buildhome/cache/.bundle’: Permission denied
mkdir: cannot create directory ‘/opt/buildhome/cache/bower_components’: Permission denied
mkdir: cannot create directory ‘/opt/buildhome/cache/.venv’: Permission denied
mkdir: cannot create directory ‘/opt/buildhome/cache/.yarn_cache’: Permission denied
mkdir: cannot create directory ‘/opt/buildhome/cache/.cache’: Permission denied
mkdir: cannot create directory ‘/opt/buildhome/cache/.cask’: Permission denied
mkdir: cannot create directory ‘/opt/buildhome/cache/.emacs.d’: Permission denied
mkdir: cannot create directory ‘/opt/buildhome/cache/.m2’: Permission denied
mkdir: cannot create directory ‘/opt/buildhome/cache/.boot’: Permission denied
mkdir: cannot create directory ‘/opt/buildhome/cache/.composer’: Permission denied
mkdir: cannot create directory ‘/opt/buildhome/cache/.gimme_cache’: Permission denied
mkdir: cannot create directory ‘/opt/buildhome/cache/.gimme_cache’: Permission denied
Installing dependencies
ls: cannot access /opt/buildhome/cache/node_version/: No such file or directory
Downloading and installing node v8.12.0...
Downloading https://nodejs.org/dist/v8.12.0/node-v8.12.0-linux-x64.tar.xz...
######################################################################## 100.0%
Computing checksum with sha256sum
Checksums matched!
Now using node v8.12.0 (npm v6.4.1)
Attempting ruby version 2.3.6, read from environment
Using ruby version 2.3.6
Using PHP version 5.6
Installing gem bundle
There was an error while trying to write to `/opt/buildhome/repo/.bundle/config`. It is likely that you need to grant
write permissions for that path.
Error during gem install
Makefile:17: recipe for target 'website' failed
make: *** [website] Error 1
```

--- a/website/Makefile
+++ b/website/Makefile
@@ -1,5 +1,5 @@
 configure-cache:
-	@mkdir -p tmp/cache
+	@mkdir -p tmp/cache .bundle
 
 build: configure-cache
 	@echo "==> Starting build in Docker..."
@@ -9,6 +9,7 @@ build: configure-cache
 		--tty \
 		--volume "$(shell pwd):/opt/buildhome/repo" \
 		--volume "$(shell pwd)/tmp/cache:/opt/buildhome/cache" \
+		--user buildbot:$(shell id -g) \
 		--env "ENV=production" \
 		netlify/build \
 		build "sh bootstrap.sh && middleman build --verbose"
@@ -21,6 +22,7 @@ website: configure-cache
 		--tty \
 		--volume "$(shell pwd):/opt/buildhome/repo" \
 		--volume "$(shell pwd)/tmp/cache:/opt/buildhome/cache" \
+		--user buildbot:$(shell id -g) \
 		--publish "4567:4567" \
 		--publish "35729:35729" \
 		--env "ENV=production" \

